src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -973,7 +1019,8 @@
         end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
         az storage file generate-sas -p path/file.txt -s MyShare --account-name MyStorageAccount --permissions rcdw --https-only --expiry $end
   - name: Generates a shared access signature for the file. (autogenerated)
```

Comment:
example name should be active voice.

Reply:
You mean `Generate` instead of `Generates` ?

Reply:
![image](https://user-images.githubusercontent.com/14357159/71961622-bb477f00-3232-11ea-95de-94ad0b9f5329.png)

I guess Aladdin learns from here

Reply:
Update manually